kolibri/core/content/api.py

```diff
@@ -1401,7 +1417,10 @@ def mean(data):
 class ContentNodeProgressViewset(
     TreeQueryMixin, viewsets.GenericViewSet, ListModelMixin
 ):
-    filter_backends = (DjangoFilterBackend,)
+
+    filter_backends = (DjangoFilterBackend, filters.OrderingFilter)
+    ordering_fields = ["last_interacted"]
+    ordering = ("lft", "id")
     filter_class = UserContentNodeFilter
```

Comment:
We should also modify this `get_queryset` method to add the annotation here instead of in the `get_resume_queryset` function.